docs/releases/1.7.txt

```diff
@@ -637,7 +637,7 @@ Forms
   value.
 
 * :attr:`SelectDateWidget.months
-  <django.forms.extras.widgets.SelectDateWidget.months>` can be used to
+  <django.forms.SelectDateWidget.months>` can be used to
```

Comment:
Keeping the link is what we usually do. The path isn't display in this case anyway.